packages/scrollable-image-ui/webpack.config.js

```diff
@@ -46,6 +47,56 @@ const config = {
             )
           },
           name: 'polyfill',
+          priority: 9,
+          enforce: true,
+        },
+        react: {
+          test: module => {
+            return (
+              module.context &&
+              /node_modules\/(react|history|redux|styled-components)/.test(
+                module.context
+              )
+            )
+          },
+          name: 'react-base',
+          priority: 10,
+          reuseExistingChunk: true,
+          enforce: true,
+        },
+        orangutan: {
+          test: module => {
+            return (
+              module.context &&
+              module.context.includes('node_modules/@twreporter/orangutan')
+            )
+          },
+          name: 'orangutan',
+          priority: 12,
+          reuseExistingChunk: true,
+          enforce: true,
```

Comment:
Do we need `orangutan` cacheGroup since we already use dynamic import to split the chunk?

Could we just add `webpackConfig.output.chunkFilename` ? (reference:  [webpack doc](https://webpack.js.org/guides/code-splitting/#dynamic-imports))